pkg/util/docker/dockerfile/builder/client.go

```diff
@@ -162,6 +167,9 @@ func (e *ClientExecutor) Build(r io.Reader, args map[string]string) error {
 			return err
 		}
 		glog.V(4).Infof("step: %s", step.Original)
+		if e.LogFn != nil {
```

Comment:
Maybe.  I wouldn't always want it to do something.

On May 3, 2016, at 1:59 PM, Michal Fojtik <notifications@github.com> wrote:

In pkg/util/docker/dockerfile/builder/client.go
<https://github.com/openshift/origin/pull/8726#discussion_r61926351>:

> @@ -162,6 +167,9 @@ func (e *ClientExecutor) Build(r io.Reader, args map[string]string) error {
>  			return err
>  		}
>  		glog.V(4).Infof("step: %s", step.Original)
> +		if e.LogFn != nil {

i think this will always be set? ;-)

—
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub
<https://github.com/openshift/origin/pull/8726/files/8315d2eecb0f67853a934bb400380bd8683305a0#r61926351>